index.js

```diff
@@ -1,480 +1,183 @@
-'use strict'
-var fs = require('fs')
-var request = require('request')
-var _ = require('highland')
+const axios = require('axios')
+const fs = require('fs')
+const logger = require('./logger')
```

Comment:
We don't seem to provide a way to change the logging level. We should do so. That might be best be done through the existing `config` function, like `wrapper.config({ logLevel: "info" })`. Or `wrapper.setLogLevel` if you think that's cleaner.